packages/utils/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { default as getWalletCurrencyIcon } from './getWalletCurrencyIcon';
 export { default as groupTransactionsByDay } from './groupTransactionsByDay';
+export { default as getWalletCurrencyIcon } from './getWalletCurrencyIcon';
```

Comment:
Not related to this PR and not sure how those ended up here 😅 But `@deriv/utils` package is a place for reusable utilities, Not using case-specific code similar to what we have in `@deriv/shared`. I'm not sure about `groupTransactionsByDay` but I think `getWalletCurrencyIcon` should be removed, The `useWalletsList` hook is already giving the same icons. 

cc: @nijil-deriv

Reply:
yes @farzin-deriv you are absolutely right. i waited for adding this icons to `useWalletsList` hook to remove `getWalletCurrencyIcon` from utils. the thing i didn't do that because we use it not only in transfer related components, but also in other places, that need to be checked by QA after refactoring i believe. but this is definitely should be done in cleanup stage